src/js/components/Form/stories/HelpAndError.js

```diff
@@ -21,6 +21,26 @@ export const HelpAndError = () => (
         />
       </FormField>
     </Form>
+    <Form>
+      <FormField
+        label="Label"
+        htmlFor="text-input"
+        help={
+          <Text weight="lighter" size="small">
+            Enter HPE email
```

Comment:
```suggestion
            Text to help the user know what is possible
```
Just want to make it clear to people that this is help text, and be a little more generic than "HPE email" for the grommet storybook audience